src/controllers/AuthController.js

```diff
@@ -1,7 +1,11 @@
+/* eslint-disable no-await-in-loop */
 /* eslint-disable consistent-return */
 const { v4: uuidv4 } = require('uuid');
 const redisClient = require('../utils/redis');
 const dbClient = require('../utils/db');
+const { Staff } = require('../models/staff');
+
+const models = ['Faculty', 'Department', 'Course', 'Project', 'Record', 'Staff', 'Student', 'Schedule', 'Record'];
```

Comment:
This is already defined in the `base` model. You should reference that instead.

_**P. S.:** You probably won't even need to do that either based on the comments below. . ._